model.py

```diff
@@ -151,7 +151,7 @@ def toggle(self, uid, user_profile):
     def render_text(self):
         title = util.make_html_bold(self.title)
         if self.people:
-            title += ' ({} {})'.format(len(self.people), util.emoji_people_unicode())
+            title += u'({} {})'.format(len(self.people), util.emoji_people_unicode())
```

Comment:
I meant the space between the count and the emoji

Reply:
Oops sorry. Updated accordingly